pkgs/tools/system/chase/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl ,pkgconfig, libatomic_ops , boehmgc }:
+
+stdenv.mkDerivation rec {
+  name = "chase_0.5.2";
+
+  buildInputs = [ pkgconfig libatomic_ops boehmgc ] ;
+  src = fetchurl {
+    url = "mirror://debian/pool/main/c/chase/${name}.orig.tar.gz";
+    sha256 = "68d95c2d4dc45553b75790fcea4413b7204a2618dff148116ca9bdb0310d737f";
+  };
+
+  doCheck = true;
+  makeFlags = [ "-e" ];
+  makeFlagsArray="LIBS=-lgc";
+
+  meta = {
```

Comment:
you could use `meta = with stdenv.lib;` here so you don't need the full qualified attributes anymore